pkg/ipam/ipam_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
```

Comment:
I don't believe this line is necessary, since the filename ends in `_windows.go`.

[[Reference](https://golang.org/pkg/go/build/#hdr-Build_Constraints)]

Reply:
removed